feature/system/src/main/java/io/github/droidkaigi/confsched2020/system/ui/viewmodel/SystemViewModel.kt

```diff
@@ -13,4 +17,20 @@ class SystemViewModel @Inject constructor() : ViewModel() {
     fun onError(error: AppError) {
         mutableErrorLiveData.value = error
     }
+
+    fun sendEventToCalendar(
+        context: Context,
```

Comment:
This should be *activity* as @takahirom has requested. This can crash applications if any non-activity context has been passed.  And also, in this case, there is no reason to create a new task by adding `FLAG_ACTIVITY_NEW_TASK` to an intent that will be created so only changing *Context* to *Activity* is acceptable.